tests/gnrc_rpl/tests/01-run.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2021 Benjamin Valentin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import subprocess
+import atexit
+import time
+
+from subprocess import Popen
+from riotctrl_shell.gnrc import GNRCICMPv6Echo, GNRCICMPv6EchoParser
+from riotctrl_shell.netif import Ifconfig
+from contextlib import ContextDecorator
+from riotctrl.ctrl import RIOTCtrlBoardFactory
+from riotctrl_ctrl import native
+from riotctrl_shell.netif import IfconfigListParser
+
+PARSERS = {
+    "ping6": GNRCICMPv6EchoParser(),
+    "ifconfig": IfconfigListParser(),
+}
+
+
+class RIOTCtrlAppFactory(RIOTCtrlBoardFactory, ContextDecorator):
```

Comment:
AFAICT the ContextDecorator is needless here (that'd be used if something said `@RIOTCtrlAppFactory(...)` before a function, but the way it's used here in `with RIOTCtrlAppFactory(...)` works without too).